desktop/apps/consignments2/client/reducers.js

```diff
@@ -0,0 +1,66 @@
+import * as actions from './actions'
+import ChooseArtist from '../components/choose_artist'
+import CreateAccount from '../components/create_account'
+import DescribeWork from '../components/describe_work'
+import UploadPhoto from '../components/upload_photo'
+import u from 'updeep'
+import { combineReducers } from 'redux'
+import { data as sd } from 'sharify'
+import { last } from 'underscore'
+import { routerReducer } from 'react-router-redux'
+
+const stepsMapping = [
+  {
+    component: CreateAccount,
```

Comment:
It concerns me a bit to have direct references to components in your reducers. Everything in the store should really just be dumb data. It seems more natural to me to represent a mapping of application state to components in your router configuration.